action.yml

```diff
@@ -42,16 +42,16 @@ runs:
       path: summary
   # output true if any state in inputs.terraform-directory changed
   - id: summary
-    if: steps.check.outputs.skip == 'false'
+    if: (success() || failure()) && steps.check.outputs.skip == 'false'
```

Comment:
Why continue if there's a failure? :thinking:

Reply:
I should make it more specific.
If the download artifacts step failed

Reply:
If the download fails, why should the action continue?